gap/PolygonalComplexes/graphs.gd

```diff
@@ -850,5 +851,49 @@ DeclareAttribute( "FaceDigraphsGraph", IsPolygonalSurface );
 DeclareAttribute( "FaceNautyGraph", IsPolygonalSurface );
 #! @EndGroup
 
-
-
+# TODO If there are more methods which construct surfaces with a given property, it is useful to have a method AllSimplicialSurfacesByFacesOfEdges which use this method.
```

Comment:
I think instead of writing TODO inside the .gd file you could create an issue or we use a separate TODO file or else it might get lost :)